backend/api/Services/RobotService.cs

```diff
@@ -20,6 +20,24 @@ public async Task<Robot> Create(Robot newRobot)
             return newRobot;
         }
 
+        public async Task<Robot> Create(string name, string host, int port)
```

Comment:
I guess this raises a question to this approach - When creating a robot from the ISAR MQTT message, we don't have enough information.
Maybe this points to the possibility that we should not automatically create robots in database, and that this should be done by users through the frontend, as it is a rare and important operation.

Reply:
You are right. Lets log an error instead of creating a new robot when an isar instance connects and there is no robot with the same name/id in the database